src/rdata.cpp

```diff
@@ -142,7 +142,6 @@ void ReturnData::initializeFullReporting(bool quadratic_llh) {
             srz.resize(nmaxevent * nz * nplist, 0.0);
         }
 
-        ssigmay.resize(nt * ny * nplist, 0.0);
```

Comment:
don't delete but only initialize if `sensi_meth == SensitivityMethod::forward`?

Reply:
That's already taken care of by `ReturnData::initializeResidualReporting` which is called a few lines above.
https://github.com/AMICI-dev/AMICI/blob/353e7f9106c6592921f191a0bc816280afdff3c0/src/rdata.cpp#L75-L86